go/consensus/cometbft/apps/keymanager/churp/txs.go

```diff
@@ -0,0 +1,253 @@
+package churp
+
+import (
+	"fmt"
+
+	beacon "github.com/oasisprotocol/oasis-core/go/beacon/api"
+	tmapi "github.com/oasisprotocol/oasis-core/go/consensus/cometbft/api"
+	churpState "github.com/oasisprotocol/oasis-core/go/consensus/cometbft/apps/keymanager/churp/state"
+	"github.com/oasisprotocol/oasis-core/go/consensus/cometbft/apps/keymanager/common"
+	"github.com/oasisprotocol/oasis-core/go/keymanager/churp"
+)
+
+func (ext *churpExt) create(ctx *tmapi.Context, req *churp.CreateRequest) error {
```

Comment:
Should creating a new churp instance require additional stake? As it seems to increase periodic processing complexity. Or is there a reasonable limit per runtime?

Reply:
Currently no limit (except `uint8`). Larger stake makes sense, but I guess I need to add an option to delete/disable an instance then.

Reply:
I will implement this latter, as I need to first include churp statuses in the genesis file and update method for calculating stake from it.